plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -294,9 +296,20 @@ public String checkAndAddRunningTask(MLTask mlTask, Integer runningTaskLimit) {
         if (error != null) {
             return error;
         }
-        if (mlCircuitBreakerService.isOpen()) {
-            mlStats.getStat(MLNodeLevelStat.ML_NODE_TOTAL_CIRCUIT_BREAKER_TRIGGER_COUNT).increment();
+        try {
+            AccessController.doPrivileged((PrivilegedExceptionAction<Void>) () -> {
+                if (mlCircuitBreakerService.isOpen()) {
```

Comment:
Can we move `AccessController.doPrivileged(` to inside of the `isOpen` method? So we don't need to add this for everywhere using `isOpen` method

Reply:
Yes I think it's doable. This is a quick change complaint with all APIs. Moving to CB level would need revisits to AccessController in other APIs. But overall the change shouldn't be big.